pkg/cli/tui/multi_select.go

```diff
@@ -0,0 +1,222 @@
+// everest
+// Copyright (C) 2025 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package tui provides UI elements for the CLI.
+package tui
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/charmbracelet/bubbles/help"
+	"github.com/charmbracelet/bubbles/key"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+)
+
+var (
+	// ------
+	// Style applied to option label in case its checkbox is selected.
+	optionStyle = lipgloss.NewStyle().
+			Foreground(
+			lipgloss.AdaptiveColor{Light: "#0E5FB5", Dark: "#93C7FF"},
+		)
+	// Style applied to option label in case its checkbox is unselected.
+	hoverOptionStyle = lipgloss.NewStyle().
+				Foreground(lipgloss.AdaptiveColor{Light: "#0B4A8C", Dark: "#62AEFF"}).
+				Background(lipgloss.AdaptiveColor{Light: "#E2EFFC", Dark: "#2F435B"})
+
+	// Selected checkbox symbol.
+	selectedCheckBox = "{X}"
+
+	// Unselected checkbox symbol.
+	unselectedCheckbox = "{ }"
```

Comment:
I'd personally prefer the original checkbox..
```suggestion
	// Selected checkbox symbol.
	selectedCheckBox = "[x]"

	// Unselected checkbox symbol.
	unselectedCheckbox = "[ ]"
```

Reply:
Me too, Nina asked to change it